File: src/main/java/gr/apt/repository/notification/NotificationRepository.java
package gr.apt.repository.notification;

import gr.apt.persistence.entity.notification.Notification;
import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase;

import javax.enterprise.context.ApplicationScoped;
import java.math.BigInteger;

@ApplicationScoped
public class NotificationRepository implements PanacheRepositoryBase<Notification, BigInteger> {
}

File: src/main/java/gr/apt/repository/notification/NotificationRepository.kt
package gr.apt.repository.notification

import gr.apt.persistence.entity.notification.Notification
import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase
import java.math.BigInteger
import javax.enterprise.context.ApplicationScoped

@ApplicationScoped
class NotificationRepository : PanacheRepositoryBase<Notification?, BigInteger?>